datasets/scripts/GOES-16-17_FireDetection

```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright 2020 The Google Earth Engine Community Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
```

Comment:
Please switch the comment from the xml style to Javascript style comments.  e.g. like
https://github.com/google/earthengine-community/blob/master/samples/javascript/guides/arrays02.js

Bugger.  The file should also end in .js.  Sorry I didn't catch that before.